scripts/tools/initialize_virtualenv.py

```diff
@@ -112,7 +112,7 @@ def main():
     Setup local virtual environment.
     """
     airflow_home_dir = Path(os.environ.get("AIRFLOW_HOME", Path.home() / "airflow"))
-    airflow_sources = Path(__file__).parents[2]
+    airflow_sources = Path(__file__).resolve().parents[2]
```

Comment:
I was thinking if `airflow_sources = Path(__file__).parents[2].resolve()` is better, but it probably does not matter either way sicne we control the `scripts/tools` directory and there’s very unlikely any non-regular entries (e.g. symlinks).